Feat: Updated Alerts

hussiantabha opened the pull request.

hussiantabha committed:
Feat: Updated Alerts

Manju-sree commented:
Everything looks fine to me with alerts component..

munjalSaksham71 reviewed (commented):
Looks Good

Priya-Patel31 commented on app.css:
```diff
@@ -158,3 +159,56 @@ a{
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    border: 0px solid black;
```
you can use border:none instead of this !!

hussiantabha replied:
Done Thanks

Priya-Patel31 commented on app.css:
```diff
@@ -158,3 +159,56 @@ a{
+}
+.alert-primary{
+    background-color: #CCE5FF;
+    color: #0064B8;
```
don't use colors directly rather than use var(--color-name) !!

Priya-Patel31 commented on app.css:
```diff
@@ -158,3 +159,56 @@ a{
+    color: #0064B8;
+}
+.alert-secondary{
+    background-color: #E2E3E5;
```
don't use colors directly rather than use var(--color-name) !!

Priya-Patel31 commented on app.css:
```diff
@@ -158,3 +159,56 @@ a{
+    color: #856404;
+}
+.alert-info{
+    background-color: #D1ECF1;
```
don't use colors directly rather than use var(--color-name) !!

Priya-Patel31 commented on app.css:
```diff
@@ -158,3 +159,56 @@ a{
+    color:#818192;
+}
+.alert-dark{
+    background-color: #D6D8D9;
```
don't use colors directly rather than use var(--color-name) !!

t007rushi commented:
Looks Good made nice color variations, one thing just add light grey border to white BG Alert else merge it

Priya-Patel31 commented on app.css:
```diff
@@ -158,3 +159,56 @@ a{
 }
+
+/* alerts */
+.alert-div{
```
make different CSS file for every component rather than combining it with app.css

hussiantabha replied:
Yes done!

Priya-Patel31 commented on app.css:
```diff
@@ -158,3 +159,56 @@ a{
+    border-radius: 0.25rem;
+    flex-wrap: wrap;
+    flex-grow: 1;
+    padding: 1rem 1rem;
```
just a suggestion can use padding : 1rem ;

hussiantabha replied:
Done Thanks!

Priya-Patel31 commented on app.css:
```diff
@@ -158,3 +159,56 @@ a{
     height: 10rem;
     border-radius: 100%;
```
there is no need of border-radius property to be used of !!

hussiantabha replied:
If I don't add this then my picture wouldn't get rounder.

hussiantabha committed:
Update: Diff component file & minor changes

Separated every components into their own css file and some removed unwanted code.
